### ISSUE TITLE
[KPI Targets & Announcements] Adds NULL values upon updating

### ISSUE THREAD

**Issue by ellaineronquillo** [rsx-labs/aide-frontend#257]
**Describe the bug**
Adds NULL values upon updating the KPI Targets / Announcements

**To Reproduce**
1. On the Dashboard, Click on one of the KPI data/Announcements
2. Do not input any information retain the previous data  then Click "Update"  a prompt window appeared stating that it was successfully updated
3. Go the updated KPI/Announcements again and you will see that another line/space is added on the field. Per checking the field is accepting a NULL VALUES and will add another line or space once the update button is  pressed

**Expected behavior**
In short, if the data is the same once the Update button is pressed, the data should be the same and will not add another space on the field

**Screenshots**
![image](https://user-images.githubusercontent.com/54298100/65937255-ea8b6a80-e451-11e9-9b58-62191079a827.png)

![image](https://user-images.githubusercontent.com/54298100/65937259-ecedc480-e451-11e9-8e77-bb44a155920f.png)

**Version**
 - Version [2.7.1]

**Comment by GiannCarlo** [comment]
I am not able to replicate this incident.

**Comment by charissasylim** [comment]
@vanchsalvador are you able to recreate this?

**Comment by charissasylim** [comment]
Fixed 2.9.0
